### DISCUSSION TITLE
status is: In mempool. It's a local status?

### DISCUSSION THREAD

**Discussion by ninvfeng** [Chia-Network/chia-blockchain#10327] [Support]
I want to know transaction status: In mempool 
It's a local status or not?
When I have two full_node, one send a transaction, but status is : In mempool . The other will show this In mempool status transaction or not?
We were stolen 1158 xch! This information is very important for us. Hope you can tell me Please!

**Comment by Dessee** [comment]
No, all countries. we need iit like to be wallet that can access to all currency

**Reply by ninvfeng** [reply]
That means if the private key is leaked and transfer elsewhere, my node exec `chia wallet get_transactions` may have transactions in the state of: In mempool, Right?
